src/esm_environment/esm_environment.py

```diff
@@ -541,7 +549,7 @@ def write_dummy_script(self, include_set_e=True):
         with open("dummy_script.sh", "w") as script_file:
             # Write the file headings
             script_file.write(
-                f'#!{self.config.get("sh_interpreter", "/bin/bash")} -l\n'
+                f'#!{self.config.get("sh_interpreter", "/bin/bash")}\n'
```

Comment:
I catch this one with `esm_tests`: This flag "makes bash to act as it has being invoked as a login shell". This means that the comp scripts and run scripts are shielded against whatever exports the user had made on their current session, ensuring that the ESM-Tools environment defined on those scripts prevail (only the environment defined by the user in the bash_profile, bash.rc... is loaded). I am afraid that this is the type of change that could result in weird errors difficult to reproduce. I would rather have the `-l` back unles there are very good reasons that I'm not aware of to remove it. What do you think @pgierz?